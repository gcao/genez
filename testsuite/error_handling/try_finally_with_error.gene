# Test finally block executes with error
(try
  (do
    (println "In try")
    (throw "Error!"))
  (catch e
    (println "Caught:" e))
  (finally
    (println "Finally executes")))