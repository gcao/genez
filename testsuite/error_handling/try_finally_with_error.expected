In try
Caught: Error!
Finally executes